website/docs/docs/build/sl-getting-started.md

```diff
@@ -100,13 +100,13 @@ Interact and test your metric using the CLI before committing it to your MetricF
 
 Follow these steps to test and query your metrics using MetricFlow:
 
-1. Make sure you install `metricflow` and [dbt adapter](/docs/supported-data-platforms) in the CLI. This is because you're installing MetricFlow as an extension of the dbt adapter. 
-Currently, the supported adapters are Snowflake and Postgres (BigQuery, Databricks, Redshift coming soon).
+1. Make sure you install the `metricflow` and [dbt adapter](/docs/supported-data-platforms) in the CLI. This is because you're installing MetricFlow as an extension of the dbt adapter. Currently, the supported adapters are Snowflake and Postgres (BigQuery, Databricks, and Redshift coming soon).
```

Comment:
```suggestion
1. Make sure you install the `metricflow` and [dbt adapter](/docs/supported-data-platforms) in the CLI using the `pip install "dbt-metricflow[your_adapter_name]"` command. This is because you're installing MetricFlow as an extension of the dbt adapter. Currently, the supported adapters are Snowflake and Postgres (BigQuery, Databricks, and Redshift coming soon).
```